### PR TITLE
IB/SEND-OPS/STR: added obj-str implementation

### PR DESCRIPTION
**Pull request by hoopoepg**
- added obj_str implementation for send-ops mpool to simplify
  resource leaks investigation

### CODE REVIEW

**Review by dmitrygx — APPROVED** [review]

File: src/uct/ib/rc/base/rc_iface.c [hunk 1]
```diff
@@ -966,3 +971,30 @@ void uct_rc_iface_vfs_refresh(uct_iface_h iface)
...
 }
+
+static void ucp_send_op_mpool_obj_str(ucs_mpool_t *mp, void *obj,
+                                      ucs_string_buffer_t *strb)
+{
+    uct_rc_iface_send_op_t *op = obj;
+    const char *handler_func_name;
+    const char *comp_func_name;
+
+    ucs_string_buffer_appendf(strb, "flags:%x ", op->flags);
```
Comment (yosefe) [inline src/uct/ib/rc/base/rc_iface.c:982]: flags:0x%x

File: src/uct/ib/rc/base/rc_iface.c [hunk 2]
```diff
@@ -966,3 +971,30 @@ void uct_rc_iface_vfs_refresh(uct_iface_h iface)
...
+
+    handler_func_name = ucs_debug_get_symbol_name(op->handler);
+    ucs_string_buffer_appendf(strb, "handler:%s()", handler_func_name);
```
Comment (yosefe) [inline src/uct/ib/rc/base/rc_iface.c:985]: maybe unify with line 982 to single appendf() call?

File: src/uct/ib/rc/base/rc_iface.c (hunk 1 above)
Comment (hoopoepg) [reply src/uct/ib/rc/base/rc_iface.c]: fixed

File: src/uct/ib/rc/base/rc_iface.c (hunk 2 above)
Comment (hoopoepg) [reply src/uct/ib/rc/base/rc_iface.c]: yep, unified

**Review by yosefe — APPROVED** [review]

File: src/uct/ib/rc/base/rc_iface.c [hunk 3]
```diff
@@ -966,3 +971,28 @@ void uct_rc_iface_vfs_refresh(uct_iface_h iface)
...
+
+    ucs_string_buffer_appendf(strb, "flags:0x%x handler:%s()", op->flags,
+                              handler_func_name);
+
+    if (op->flags & UCT_RC_IFACE_SEND_OP_STATUS) {
+        ucs_string_buffer_appendf(strb, " status:%d", op->status);
+    }
+
+    if (op->user_comp != NULL) {
+        comp_func_name = ucs_debug_get_symbol_name(op->handler);
```
Comment (brminich) [inline src/uct/ib/rc/base/rc_iface.c:990]: did you mean to add `user_comp->func`?

Comment (hoopoepg) [reply src/uct/ib/rc/base/rc_iface.c]: copy-paste issue

**Review by brminich — APPROVED** [review]

**Review by yosefe — APPROVED** [review]

**Comment by hoopoepg** [comment]
@yosefe back it port into int3?

**Comment by yosefe** [comment]
> @yosefe back it port into int3?

yes

**Comment by hoopoepg** [comment]
timeout

**Comment by hoopoepg** [comment]
bot:pipe:retest

**Pull request merged**